knowledge-base/common-animations-decrease.md

```diff
@@ -1,83 +1,108 @@
 ---
-title: Remove animations of dropdowns or make them faster
+title: Remove DropDown Animations or Make Them Faster
 description: How to remove the dropdown animations or increase their speed
```

Comment:
We really heavily on search to get this better indexed (including internally). I would invest some time in improving the page title and description. For example
```suggestion
description: How to remove the DropdDown for Blazor animations or increase their speed. See an example of a CSS override handling the performance and accessibility scenario.
```